examples/custom_theme_example/README.md

```diff
@@ -0,0 +1,5 @@
+# custom_theme_example
+
+An example of how to use custom theme.
```

Comment:
This is not proper English. You are missing an `a` before `custom theme`. The sentence should be 

> An example of how to use **a** custom theme.

Reply:
I'm using Grammarly for spelling and grammar checking. Unfortunately, it does not work in VS Code but it helped me with improving my grammar.